hawkular-inventory-impl-cassandra/src/main/java/org/hawkular/inventory/impl/cassandra/Statements.java

```diff
@@ -36,6 +39,10 @@
     static final String TARGET_CP = "target_cp";
     static final String NAME = "name";
     static final String PROPERTIES = "properties";
+    static final Map<String, String> RELATIONSHIP_MAIN_CP = ImmutableMap.<String, String>builder()
```

Comment:
This I think becomes unnecessary in the latest version of the model where we have 3 relationship tables.